news/2022-05-08-aptb-fanart-results.md

```diff
@@ -0,0 +1,109 @@
+date: 2022-05-08 11:00:00 +0000
+---
+
+The results are in, so come check and see whether your vote has found its place to belong on the main menu!
+
+![](https://assets.ppy.sh/contests/142/banners/Dreamxiety.jpg)
+
+Boldly striding forward into one of our broadest themes yet, **135** artists made the brave step to put their work up for the consideration of the masses.
+
+And considering is exactly what the masses did! With only **14** votes to wield, it was a tough set of choices to make, but over nine thousand (nice) of you let loose your stars of judgement, and so, we have finalists to crown.
+
+Let's get straight into the results - but not before a quick reminder from yours truly about how you can click any of the finalist banners to be directed to a high-quality version of the image for all your wallpapering needs.
+
+Got that? Onwards:
+
+[![](https://assets.ppy.sh/contests/142/banners/Dreamxiety.jpg)](https://assets.ppy.sh/contests/142/winners/Dreamxiety.png)
+
+Taking top spot with pippi, Mocha and Chirou enjoying a scenic bike ride through a springtime park is none other than two-time (now three) finalist [**Dreamxiety**](https://osu.ppy.sh/users/13103233) with a whopping **3713** votes!
+
+An absolutely stunning piece, well-deserving of the big one!
+
+[![](https://assets.ppy.sh/contests/142/banners/utaaa.jpg)](https://assets.ppy.sh/contests/142/winners/utaaa.png)
+
+Showcasing another springtime spectacle, [**utaaa**](https://osu.ppy.sh/users/9315038)'s academic artistry portrays pippi and Mocha in a schooltime setting to the approval of **3270** voters.
+
+Pencil dog and a mysterious frog both make a low-key cameo appearance, of course.
+
+[![](https://assets.ppy.sh/contests/142/banners/Llamartist.jpg)](https://assets.ppy.sh/contests/142/winners/Llamartist.png)
+
+The very aptly named [**Llamartist**](https://osu.ppy.sh/users/14697690) impressed **2938** voters with a beautiful bout of pippi and Mocha appreciating aquatic aesthetics. 
+
+Is it a date? A field trip for a school project? Are we the fish Mocha is pointing at? So many questions, but we'll thankfully have the next few months to think it over while this wonderful piece makes its well-deserved main menu appearance.
+
+[![](https://assets.ppy.sh/contests/142/banners/DGen.jpg)](https://assets.ppy.sh/contests/142/winners/DGen.png)
+
+[**DGen**](https://osu.ppy.sh/users/13290907) won **2813** votes with a rendition of pippi and Chirou enjoying a vast assortment of Japanese foods, complete with "Choked Top Play Soy Sauce (with Extra Salt)", presumably freshly harvested from your favourite osu! streamer or top player of choice.
+
+[![](https://assets.ppy.sh/contests/142/banners/PacifiedPaints.jpg)](https://assets.ppy.sh/contests/142/winners/PacifiedPaints.png)
+
+The very un-aptly named [**PacifiedPaints**](https://osu.ppy.sh/users/8377453) produced a punchy punk picture of pippi and Mocha engaging in some urban art to the approval of **2727** voters.
```

Comment:
[it's a word](https://www.thefreedictionary.com/unaptly)

```suggestion
The very unaptly named [**PacifiedPaints**](https://osu.ppy.sh/users/8377453) produced a punchy punk picture of pippi and Mocha engaging in some urban art to the approval of **2727** voters.
```

<img src="https://wala.s-ul.eu/B4nAVTV2" width="70" />